index.html

```diff
@@ -203,7 +203,7 @@ <h1 class="collapse">Discounted Dining Finder: Places to Eat Out To Help Out</h1
 		let fsa = "Not found / rated";
 		if (d.fsa > 0) {
 
-		    fsa = "<img src='fhrs_" + d.fsa + "_en-gb.jpg' alt='Food Standard Agency Score: " + d.fsa + "' />";
+		    fsa = "<img src='fhrs_md_" + d.fsa + "_en-gb.jpg' alt='Food Standard Agency Score: " + d.fsa + "' />";
```

Comment:
AMBER: I was looking at the ratings yesterday and I think we might have to pass the RatingsKey instead of the RatingValue - as Scotland has got a different rating system, so if you go to Glasgow none of the ratings work at the moment.

Reply:
That's annoying. I'll re generate the data. 

Great spot!

Reply:
I had a quick look at this tonight. The rating value should be ok so data is fine as is. I do need to tweak the script here though. I'll have time tomorrow to take a proper look though

Reply:
Ive pushed my changes to https://eat.exussum.co.uk Scotland seems to be better. Stirling is still broken as the whole address is in "address line 1" eg `<AddressLine1>5 Dumyat Road Causewayhead Stirling FK9 5HA</AddressLine1>`

Reply:
Fixed the sterling issue too